### ISSUE TITLE
Search Feature in failed jobs

### ISSUE THREAD

**Issue by Gauravpurohit409** [resque/resque#1889]
Recently I encountered a scenario where jobs were failing, but I was not able to find them as there were 1200 failed jobs. 

So I am thinking if we would have a search option in it, it would be very helpful, should we do it?

**Comment by PatrickTulskie** [comment]
Usually when I find a big backlog of errors I wind up going into the console to clean them out. I also use resque-cleaner to help with those situations. I'm not against a search or lookup interface being added to resque-web.

**Comment by Gauravpurohit409** [comment]
Yes but if a product is in production then I don't feel like opening console and do thinks as it is somewhat risky. so if we have search option it will be very useful
